config/kubernetes-sigs/sig-k8s-infra/teams.yaml

```
@@ -9,6 +9,15 @@ teams:
     - thockin
     privacy: closed
   community-images-writers:
+    description: write access to community-images
+    maintainers:
+      - ameukam
+    members:
+      - dims
+      - spiffxp
+      - thockin
+    privacy: closed
```

```suggestion
    description: write access to community-images
    maintainers:
      - ameukam
    members:
      - dims
      - spiffxp
      - thockin
    privacy: closed
    previously:
      - community-image-writers
```
You can rename the group to maintainers by adding the previously key and then the previous name of the group as a list item.